src/nvim/main.c

```diff
@@ -547,10 +547,12 @@ int main(int argc, char **argv)
   if (params.n_commands > 0)
     exe_commands(&params);
 
+  /* Must come before the may_req_ calls. */
```

Comment:
We don't have any of these `may_req_*` functions.

Either they weren't merged yet, or they're not applicable. But apparently they existed in 2013 already, so it's highly likely that this patch is N/A. (We probably handle this kind of stuff in `src/nvim/tui/tui.c`.)

@justinmk Correct?

Reply:
Yes, N/A, so the comment can be deleted.

Reply:
Ok, thank you, I saw that some code was different in the neovim source but decided to make the change just in case

Reply:
I'll delete the comment when merging. Just waiting for CI to finish.